src/v/kafka/protocol/schemata/BUILD

```diff
@@ -0,0 +1,181 @@
+load("@rules_python//python:defs.bzl", "py_binary")
+load("//src/v/kafka/protocol/schemata:generator.bzl", "redpanda_cc_kafka_message_library")
+
+py_binary(
+    name = "generator",
+    srcs = ["generator.py"],
+    main = "generator.py",
+    visibility = ["//visibility:public"],
+    deps = [
+        "@python_deps//jinja2",
+        "@python_deps//jsonschema",
+    ],
+)
+
+redpanda_cc_kafka_message_library(name = "add_offsets_to_txn_request")
```

Comment:
Massive. Do you want to hide all this in a macro so you can write a for loop over a list? Otherwise in top level BUILD files you can do for comprehensions:

```
[
  redpanda_cc_kafka_message_library(x)
  for x in [
    "add_offsets_to_txn_request",
    ...
  ]
]
```